cli/src/pcluster/schemas/cluster_schema.py

```diff
@@ -170,6 +170,21 @@ def make_resource(self, data, **kwargs):
         return RootVolume(**data)
 
 
+class LoginNodesRootVolumeSchema(BaseSchema):
+    """Represent the RootVolume schema for the LoginNodes."""
+
+    size = fields.Int(metadata={"update_policy": UpdatePolicy.LOGIN_NODES_STOP})
+    encrypted = fields.Bool(metadata={"update_policy": UpdatePolicy.LOGIN_NODES_STOP})
+    volume_type = fields.Str(metadata={"update_policy": UpdatePolicy.LOGIN_NODES_STOP})
+    iops = fields.Int(metadata={"update_policy": UpdatePolicy.LOGIN_NODES_STOP})
+    throughput = fields.Int(metadata={"update_policy": UpdatePolicy.LOGIN_NODES_STOP})
+
+    @post_load
+    def make_resource(self, data, **kwargs):
+        """Generate resource."""
+        return RootVolume(**data)
+
+
```

Comment:
This is a big change because it will impact the UX.
We do not want to have this public facing change (at least without the approval of the PM).

Reply:
Done! Thank you for your suggestion, Nicola!